app/src/main/java/org/fossasia/openevent/adapters/FilterAdapter.java

```diff
@@ -0,0 +1,113 @@
+package org.fossasia.openevent.adapters;
+
+/**
+ * Created by user on 2/29/2016.
+ */
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.CheckBox;
+import android.widget.TextView;
+
+
+import org.fossasia.openevent.R;
+import org.fossasia.openevent.data.Information;
+import org.fossasia.openevent.utils.SaveFilters;
+
+import java.util.Collections;
+import java.util.List;
+
+@SuppressWarnings("deprecation")
+/**
+ * Created by user on 02-06-2015.
+ */
+public class FilterAdapter extends RecyclerView.Adapter<FilterAdapter.MyViewHolder> {
+    private LayoutInflater inflater;
```

Comment:
You don't really require a different adapter for this. You can us the existing one.